airflow-core/tests/unit/api_fastapi/conftest.py

```diff
@@ -117,6 +117,7 @@ def create_test_client(apps="all"):
 @pytest.fixture
 def configure_git_connection_for_dag_bundle(session):
     # Git connection is required for the bundles to have a url.
+    clear_db_connections(False)
     connection = Connection(
```

Comment:
```suggestion
    clear_db_connections(False)
    # Git connection is required for the bundles to have a url.
    connection = Connection(
```

Nit (keeping the comment directly above the relevant code)